overlord/snapstate/snapstate.go

```diff
@@ -1815,7 +1815,7 @@ func keys[K comparable, V any](m map[K]V) []K {
 // If the filter returns true, the update for that snap proceeds. If
 // it returns false, the snap is removed from the list of updates to
 // consider.
-type updateFilter func(*snap.Info, *SnapState) bool
+type updateFilter = func(*snap.Info, *SnapState) bool
```

Comment:
what's the reason for this change?

Reply:
This enables mocking the functions that accept this type as a parameter.